src/test/java/spoon/reflect/visitor/CtScannerTest.java

```diff
@@ -271,9 +271,9 @@ public void exit(CtElement o) {
 		// this is a coarse-grain check to see if the scanner changes
 		// no more exec ref in paramref
 		// also takes into account the comments
-		assertEquals(3655, counter.nElement + countOfCommentsInCompilationUnits);
-		assertEquals(2435, counter.nEnter + countOfCommentsInCompilationUnits);
-		assertEquals(2435, counter.nExit + countOfCommentsInCompilationUnits);
+		assertEquals(3631, counter.nElement + countOfCommentsInCompilationUnits);
+		assertEquals(2423, counter.nEnter + countOfCommentsInCompilationUnits);
+		assertEquals(2423, counter.nExit + countOfCommentsInCompilationUnits);
```

Comment:
Just as a comment, I have no clue why we have a test case where we simply change numbers only for the CI :D

Reply:
Yea, this one is a bit weird. Maybe a `Map<Class<? extends CtElement> , int>` would be better? Then you can see which part of the AST you changed and whether that was intentional?